hardhat.config.js

```diff
@@ -1,6 +1,13 @@
-require("@nomicfoundation/hardhat-chai-matchers");
-require("@nomiclabs/hardhat-ethers");
-require("@openzeppelin/hardhat-upgrades");
+require('@nomicfoundation/hardhat-chai-matchers')
+require('@nomiclabs/hardhat-ethers')
+require('@openzeppelin/hardhat-upgrades')
+const {
+  OPERATOR_ID_A,
+  OPERATOR_KEY_A,
+  HEX_PRIVATE_KEY_A,
+  HEX_PRIVATE_KEY_B,
```

Comment:
Are these two env vars new then?  Because if it was me I'd stick `_EDCSA_` right in there in the name.  But no need to rework it at this point.